safety/auth/server.py

```
@@ -231,13 +229,17 @@ def handle_timeout(self) -> None:
             server_thread = threading.Thread(target=server.handle_request)
             server_thread.start()
-            message = f"If the browser does not automatically open in 5 seconds, " \
-                        "copy and paste this url into your browser:"
+            message = f"If the browser does not automatically open in 5 seconds, copy and paste this url into your browser:"
 
         target = uri if headless else f"{uri}&port={PORT}"
-        console.print(f"{message} [link={target}]{target}[/link]")
-        console.print()
 
+        if is_jupyter_notebook():
+            console.print(f"1")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)